### PR TITLE
Documentation for logo and design

### PR DESCRIPTION
**Pull request by nicolaswurtz**

### CODE REVIEW

File: static/logos/logo_white.svg.black
[the remark this answers is not in the document]
Comment (nicolaswurtz) [reply static/logos/logo_white.svg.black:1]: Such a good idea !

[the remark this answers is not in the document]
Comment (nicolaswurtz) [reply static/logos/logo_white.svg.black:1]: Corrected !

File: content/docs/guides/logo/index.fr.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: "Le logo"
+linkTitle: "Le logo"
+weight: 10
+description: Le logo d'OSRD, ses variantes, et son usage
+resources:
+- src: "**.svg"
+---
+
+Vous pouvez télécharger chaque logo indépendamment en cliquant directement dessus, ou <a href="pictures/logo-osrd.zip">tous les logos compressés dans un fichier zip.</a>
```
Comment (woshilapin) [inline content/docs/guides/logo/index.fr.md:10]:
```suggestion
Vous pouvez télécharger chaque logo indépendamment en cliquant directement dessus, ou <a href="pictures/logo-osrd.zip">tous les logos compressés dans un fichier zip</a>.
```

File: content/docs/guides/logo/index.fr.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Il est conseillé de bien choisir le logo à utiliser en fonction du fond sur lequel vous voulez l'afficher.
+
+La modification, ajout ou suppression de l'**ombrage** autre que tel que présenté dans les logos **ne sont pas autorisés** (cela est valable plus globalement dans tout le design, le choix de mettre des ombres portées fait partie des réflexions design, ce n'est pas un élément variable).
```
Comment (woshilapin) [inline content/docs/guides/logo/index.fr.md:14]:
```suggestion
La modification, ajout ou suppression de l'**ombrage** autre que tel que présenté dans les logos **ne sont pas autorisés** (cela est valable plus globalement dans tout le design, le choix de mettre des ombres portées fait partie des réflexions de design, ce n'est pas un élément variable).
```

**Review by woshilapin — APPROVED** [review]
Only minor and opinionated comments. Looks good to me overall.

File: content/docs/guides/logo/index.en.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+title: "Logo"
+linkTitle: "Logo"
+weight: 10
+description: The OSRD logo, its variants, and its use
+resources:
+- src: "**.svg"
+---
+
+You can download each logo independently by clicking directly on it, or <a href="pictures/logo-osrd.zip">all the logos compressed into a zip file.</a>
```
[the remark this answers is not in the document]
Comment (nicolaswurtz) [reply content/docs/guides/logo/index.en.md:10]: Done

File: content/docs/guides/logo/index.en.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+It is advisable to choose the logo you want to use carefully, depending on the background on which you want to display it.
```
[the remark this answers is not in the document]
Comment (nicolaswurtz) [reply content/docs/guides/logo/index.en.md:12]: Done

File: content/docs/guides/logo/index.fr.md (hunk 1 above)
Comment (nicolaswurtz) [reply content/docs/guides/logo/index.fr.md:10]: Done

File: content/docs/guides/logo/index.fr.md (hunk 2 above)
Comment (nicolaswurtz) [reply content/docs/guides/logo/index.fr.md:14]: done

File: content/docs/guides/design/index.en.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: "OSRD's design"
+linkTitle: "OSRD's design"
+weight: 10
+description: Colours, fonts, uses...
```
Comment (axrolld) [inline content/docs/guides/design/index.en.md:5]: To match french version
```suggestion
description: Colours, fonts, logos, uses...
```

**Review by axrolld — APPROVED** [review]
Just a tiny adjustment, not even necessary but nice to fix i'd say.

File: content/docs/guides/design/index.en.md (hunk 5 above)
Comment (nicolaswurtz) [reply content/docs/guides/design/index.en.md:5]: Thx! Corrected but in another way: I removed "logo" on the french side. We speak about logos inside another file ;)

Comment (axrolld) [reply content/docs/guides/design/index.en.md:5]: LGTM, thanks !

**Review by axrolld — APPROVED** [review]

**Pull request merged**